osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/DlmsHelper.java

```diff
@@ -120,7 +119,7 @@ public DataObject getAttributeValue(final DlmsConnectionManager conn, final Attr
      * number of attribute addresses provided.
      */
     public List<GetResult> getAndCheck(final DlmsConnectionManager conn, final DlmsDevice device,
-                                       final String description, final AttributeAddress... params) throws ProtocolAdapterException {
+            final String description, final AttributeAddress... params) throws ProtocolAdapterException {
```

Comment:
``AttributeAddress`` is een array; ik zou hier dan ook meervoud gebruiken in de naam.

Dus:

```suggestion
            final String description, final AttributeAddresses... params) throws ProtocolAdapterException {
```